CMakeLists.txt

```diff
@@ -78,17 +82,17 @@ if(BUILD_EXAMPLES)
   endif()
 endif()
 
-if(BUILD_BENCHMARKS)
+if(GPUTREESHAP_BUILD_BENCHMARKS)
```

Comment:
Renaming this variable also fixes the problem, because cuML devcontainers will no longer build GPUTreeShap benchmarks unless `-DGPUTREESHAP_BUILD_BENCHMARKS=ON` is specified. Previously this was taking `-DBUILD_BENCHMARKS` from a higher CMake invocation (building cuML and other RAPIDS repositories) and applying it transitively to GPUTreeShap.